spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/autoconfig/JpaEventStoreAutoConfiguration.java

```diff
@@ -0,0 +1,47 @@
+package org.axonframework.springboot.autoconfig;
```

Comment:
Don't forget to add this to the `spring.factories` file!
If it isn't there, your build should fail atm.

Reply:
Thank you for the reminder!